src/stream/src/task/barrier_manager/tests.rs

```diff
@@ -109,10 +121,15 @@ async fn test_managed_barrier_collection_before_send_request() -> StreamResult<(
     // Collect a barrier before sending
     manager.collect(extra_actor_id, &barrier);
 
-    // Send the barrier to all actors
-    manager
-        .send_barrier(barrier.clone(), actor_ids_to_send, actor_ids_to_collect)
+    actor_op_tx
+        .send_and_await(|tx| LocalActorOperation::InjectBarrier {
```

Comment:
Can we still extracting it as a method? Same for `AwaitEpochCommited`.

Reply:
done